src/mars_patcher/locations.py

```diff
@@ -86,7 +89,11 @@ def __init__(
         self.item_messages = item_messages
 
 
+@dataclass(frozen=True)
 class ItemMessages:
+    item_messages: frozendict[Language, str]
+    centered: bool
+
     LANG_ENUMS = {
```

Comment:
if this dict is gonna stay around, you should explicitly mark it as a `ClassVar[dict[str, Language]]` so that it's handled correctly in the dataclass code